cli/tests/integrations/test_auth.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from dcos import constants
+
+import pytest
+
+from .common import assert_command
+
+
+@pytest.fixture
+def env():
+    r = os.environ.copy()
+    r.update({
+        constants.PATH_ENV: os.environ[constants.PATH_ENV],
+        constants.DCOS_CONFIG_ENV: os.path.join("tests", "data", "dcos.toml"),
+    })
+
+    return r
+
+
+def test_info():
+    stdout = b'Authenticate to DCOS cluster\n'
+    assert_command(['dcos', 'auth', '--info'],
+                   stdout=stdout)
+
+
+def test_version():
+    stdout = b'dcos-auth version SNAPSHOT\n'
+    assert_command(['dcos', 'auth', '--version'],
+                   stdout=stdout)
+
+
+def test_login_no_auth():
+    stdout = b'Login successful!\n'
+    assert_command(['dcos', 'auth', 'login'], stdout=stdout)
+
+
+def test_logout_no_token(env):
+    assert_command(['dcos', 'auth', 'logout'], env=env)
+
+
+def test_logout_with_token(env):
+    stdout = b"[core.dcos_acs_token]: set to 'faketoken'\n"
```

Comment:
We should not print the token to the screen. Does your other PR around this cover this case?